packages/edit-site/src/components/template-details/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { getTemplateInfo } from '../../utils';
+
+export default function TemplateDetails( { template } ) {
+	if ( ! template ) {
+		return null;
+	}
+	const { title, description } = getTemplateInfo( template );
+
+	return (
+		<div className="edit-site-template-details">
+			<p className="edit-site-template-details__heading">
+				{ __( 'Template details' ) }
+			</p>
+			<p>{ `${ __( 'Name' ) }: ${ title }` }</p>
+			<p>{ `${ __( 'Description' ) }: ${ description }` }</p>
```

Comment:
If the description is missing it prints `undefined`.
I guess we could either hide the entire description line, or add an empty notice, e.g. "Description: not provided" or something like that.

Reply:
I added a default description/title to `getTemplateInfo`

Reply:
Actually, I went ahead and avoided rendering this when the info doesn't exist.